README.md

```diff
@@ -1,22 +1,50 @@
+
+### Intro
+
+The app team works on a cluster strategically located in the `East US 2` region as this is where most of their customer base can be found. They have been operating this solo AKS cluster for a quite some time [following the recommended baseline practices to work with a single AKS cluster, and they learned them all by heart](https://github.com/mspnp/aks-secure-baseline).
+
+Thanks to baseline the cluster is already available from different _Zones_ within the same region, now they realize that provided `East US 2` went fully down, they are out of game. Even though the SLA(s) are acceptable for their business continuity plan, they are starting to think what are the options, and how their stateless application (Application ID: a0042) could increase its availability in case of an incident of the magnitude mentioned above. They started conversations with the business unit (BU0001) to increment the number of clusters by one. In other words, they are proposing to go over a Multi Cluster infrastructure solution in which multiple intances of the same application could live.
+
+This architectural decision will have multiple implications for the Contoso Bicycle organization. It is not just about repeating everything they have done before but planning how efficiently share Azure resources as well as detect those that need to be added, how they are going to deploy more than one cluster as well as operate them, decide to which specific regions they deploy, and many more considerations striving for High Availability.
+
+### Federation
+
+The business unit (BU0001) approves the creation of a second cluster that could help balacing the traffic but mainly to serve as a hot backup, they are a bit worried on the required engineering effort though. The same application (Application Id: a0042) is about to span into multiple clusters, so there is a desire to find out a good mechanism for its configuration management. With that in mind, the app team is looking at what _Federation_ approaches they could follow to run different intances of the exact same app in different clusters.
+
+They know that at this point things must be kept simply, in fact they could run these two application instances from these two clusters with just a bunch of useful scripts. But they want to be sure that the selected approach is not going to be adding impediments that could prevent from scaling out their fleet of clusters down to road if there was a requirment to scale up the amount application instances.
+
+Depending on how federation is implemetned it could open a door in which a single command execution has an instant ripple effect into all your clusters. While running clusters separately like silos could keep you safe from it, the cost could be really hight by struggling at scaling the number of clusters in the future.
```

Comment:
```suggestion
Depending on how federation is implemented it could open a door in which a single command execution has an instant ripple effect into all your clusters. While running clusters separately like silos could keep you safe from the same, but the cost could be really high to scale the number of clusters in the future.
```